_layouts/page.html

```diff
@@ -9,11 +9,9 @@
 <div id="main-content" class="usa-layout-docs usa-section">
   <div class="grid-container">
     <div class="grid-row grid-gap">
-      {% if page.sidenav == true %}
       {% include sidenav.html %}
```

Comment:
I think that we'll want to have this conditional left in. It is something set in the page.md files and I don't see any reason to remove it. Although maybe I'm missing something. 

```
      {% if page.sidenav == true %}
       {% include sidenav.html %}
       {% endif %}
```

Reply:
I added the conditionals back